shell/packages/sandstorm-permissions/permissions.js

```diff
@@ -15,6 +15,18 @@
 // limitations under the License.
 
 var Crypto = Npm.require("crypto");
+var Url = Npm.require("url");
+
+SandstormPermissions = function (db) {
+  function cleanupSelfDestructing() {
+    var now = new Date();
+    db.collections.apiTokens.remove({expiresIfUnused: {$lt: now}});
+  };
+
+  // Make self-destructing tokens actually self-destruct, so they don't
+  // clutter the token list view.
+  SandstormDb.periodicCleanup(5 * 60 * 1000, cleanupSelfDestructing);
```

Comment:
Hmm, it worries me that if you constructed two SandstormPermissions objects you'd end up with multiple cleanups registered. Maybe do this is a Meteor.startup() instead?